src/modules/markets/components/markets-view.jsx

```diff
@@ -66,6 +68,14 @@ export default class MarketsView extends Component {
   }
 
   componentWillUpdate(nextProps, nextState) {
+    const newTags = decodeURIComponent(parseQuery(nextProps.location.search)[TAGS_PARAM_NAME] || '')
+    if (newTags.length) {
+      const marketsFilteredByTags = filterByTags(newTags.split(QUERY_VALUE_DELIMITER), nextProps.markets)
```

Comment:
Per my comments on `filterByTags`, you’ll need to update handling of this value accordingly.